TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/TrafficReplayerTest.java

```diff
@@ -149,10 +154,36 @@ public void testReader() throws IOException, URISyntaxException {
         Assertions.assertTrue(byteArrays.stream().allMatch(ba->ba.size()==2));
     }
 
+    @Test
+    public void testMissingStreamCausesWarning() throws URISyntaxException, IOException, ExecutionException, InterruptedException {
```

Comment:
Could you highlight where the missing stream gets introduced? It wasn't immediately obvious to me where we had cut out or removed a traffic stream then saw we create a traffic stream with a higher chunk number